test/test_e2e.rb

```diff
@@ -12,9 +12,9 @@ def test_suite_with_legit_failures
       "./spec/bar_spec.rb[1:2]",
     ], queue
 
-   assert_equal 3+RSpecQ::MAX_REQUEUES, queue.example_count
+   assert_equal 3 + RSpecQ::DEFAULT_MAX_REQUEUES, queue.example_count
 
-   assert_equal({ "./spec/bar_spec.rb[1:2]" => "3" }, queue.requeued_jobs)
+   assert_equal({ "./spec/bar_spec.rb[1:2]" => RSpecQ::DEFAULT_MAX_REQUEUES.to_s }, queue.requeued_jobs)
```

Comment:
In principle, I'd prefer to avoid using constants in assertions in order to avoid scenarios where the wrong thing is tested and to make the tests cases more concrete and readable.

Reply:
Didn't think about it like that, makes sense !